content/docs/replication/migrating-volumes.md

```diff
@@ -0,0 +1,145 @@
+
+Newly created PV (`test-pv-to-powermax-replication` in our example) is available for consumption via static provisioning by any PVC that will request it.
+
+
+## Namespace Considerations For Replication
+
+Replication Groups in CSM Replication can be made namespaced, meaning that one SC will generate one Replication Group per namespace, this is also important when migrating volumes from/to replcation storage class.
+
+When just setting one annotation `migration.storage.dell.com/migrate-to` migrated volume is assumed to be used in same namespace as original PV and it's PVC and in case of being migrated to replication enabled storage class will be inserted in namespaced Replication Group inside PVC namespace. 
+
+However, you can define in which namespace migrated volume must be used after migration by setting `migration.storage.dell.com/namespace`, you can use same annotation in scenario where you only have statically provisioned PV, don't have it bound to any PVC and want to migrate it to another storage class
+
+
+## Non Disruptive Migration
+
+You can migrate your PVs without disrupting workflows if you use StatefulSet with multiple replicas to deploy application. 
+
+Instruction (you can also use `repctl` for convenience):
+
+1. Find every PV for your StatefulSet and patch it with `migration.storage.dell.com/migrate-to` annotation that points to new storage class
+```shell
+kubectl patch pv <pv-name> -p '{"metadata": {"annotations":{"migration.storage.dell.com/migrate-to":"powermax-replication"}}}'
+```
+
+2. Ensure you have a copy of StatefulSet manifest somewhere ready, we will need it later. If you don't have it, you can get it from cluster
+```shell
+kubectl get sts <sts-name> -n <ns-name> -o yaml > sts-manifest.yaml
+```
+
+3. To not disrupt any workflows we will need to delete StatefulSet without deleting any pods, to do so you can use `--cascade` flag
+```shell
+kubectl delete sts <sts-name> -n <ns-name> --cascade=orphan
+```
+
+4. Change StorageClass in your manifest of StatefulSet to point to a new storage class, then apply it to the cluster
+```shell
+kubectl apply -f sts-manifest.yaml
+```
+
+5. Find a PVC and pod of one replica of StatefulSet delete PVCs first and Pod after it
+```shell
+kubectl delete pvc <pvc-name> -n <ns-name>
+```
+```shell
+kubectl delete pod <pod-name> -n <ns-name>
+```
+
+Wait for new pod to be created by StatefulSet, it should create new PVC that will use migrated PV. 
+
+6. Repeat step 5 until all replicas use new PVCs
+
+
+## Using repctl
+
+You can use `repctl` CLI tool to help you simplify running migration specific commands.
+
+### Single PV
+
+In most basic form repctl can do the same as kubectl, for example, migrating single PV from our example will look like: 
+
+```shell
+./repctl migrate pv test-pv --to-sc powermax-replication
+```
+
+`repctl` will go and patch the resource for you, also you can provide `--wait` flag for it to wait until migrated PV is created in cluster. 
```

Comment:
Replace comma after "you" with a period. Replace "also you can" with "You can also"